deploy/kubernetes/deployment.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       serviceAccount: csi-provisioner
       containers:
         - name: csi-provisioner
-          image: gcr.io/k8s-staging-sig-storage/csi-provisioner:canary
+          image: registry.k8s.io/k8s-staging-sig-storage/csi-provisioner:canary
```

Comment:
There is no such image?

k8s-staging-sig-storage does not get mirrored through registry.k8s.io.

Reply:
So should I remove k8s-staging part and also check the image tag